presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -503,6 +518,21 @@ public static long floorFloat(@SqlType(StandardTypes.REAL) long num)
         return floatToRawIntBits((float) floor(intBitsToFloat((int) num)));
     }
 
+    @Description("inverse of Binomial cdf given numberOfTrials, successProbability parameters and p")
+    @ScalarFunction
+    @SqlType(StandardTypes.INTEGER)
```

Comment:
why not BIGINT?

Reply:
number of trials is an integer, so how the result can be larger than an integer? (if you meant why at all integer rather than big-int, I guess that I just mimicked the types of the underline java implementation)

Reply:
Interesting. I wonder if it can handle BIGINT or only int. Do you happen to know how to check that?

Reply:
Update: after checking, it seems that Java's int is 4 bytes (like Presto's INTEGER) - so what you did is good.